[0m10:46:46.783447 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7c38ad102810>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7c38ac7cc830>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7c38ac553ef0>]}


============================== 10:46:46.786638 | d6651cb1-485f-4a4e-a525-61894cd05344 ==============================
[0m10:46:46.786638 [info ] [MainThread]: Running with dbt=1.10.13
[0m10:46:46.787141 [debug] [MainThread]: running dbt with arguments {'empty': 'None', 'partial_parse': 'True', 'write_json': 'True', 'version_check': 'True', 'printer_width': '80', 'no_print': 'None', 'log_format': 'default', 'quiet': 'False', 'profiles_dir': '/home/joyce/.dbt', 'log_path': 'logs', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'send_anonymous_usage_stats': 'True', 'invocation_command': 'dbt init dbt_cobaia_project', 'fail_fast': 'False', 'use_experimental_parser': 'False', 'cache_selected_only': 'False', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'static_parser': 'True', 'target_path': 'None', 'warn_error': 'None', 'debug': 'False', 'use_colors': 'True', 'introspect': 'True'}
[0m10:46:46.796875 [debug] [MainThread]: Starter project path: /home/joyce/joyce-saas/meu-dbt-cobaia/venv/lib/python3.12/site-packages/dbt/include/starter_project
[0m10:46:46.799077 [info ] [MainThread]: 
Your new dbt project "dbt_cobaia_project" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m10:46:46.799534 [info ] [MainThread]: Setting up your profile.
[0m10:50:49.862866 [info ] [MainThread]: Profile dbt_cobaia_project written to /home/joyce/.dbt/profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m10:50:49.869661 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 243.13684, "process_in_blocks": "2088", "process_kernel_time": 0.106366, "process_mem_max_rss": "121480", "process_out_blocks": "80", "process_user_time": 1.411859}
[0m10:50:49.870457 [debug] [MainThread]: Command `dbt init` succeeded at 10:50:49.870335 after 243.14 seconds
[0m10:50:49.870879 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7c38aca78e30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7c38aaf81910>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7c38ab364530>]}
[0m10:50:49.871297 [debug] [MainThread]: Flushing usage events
[0m10:50:50.837668 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m10:51:27.814664 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fa1b278f410>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fa1b2764ce0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fa1b2649d30>]}


============================== 10:51:27.817644 | 34628e7b-638b-4a24-9bb0-bd73c357dbb9 ==============================
[0m10:51:27.817644 [info ] [MainThread]: Running with dbt=1.10.13
[0m10:51:27.818191 [debug] [MainThread]: running dbt with arguments {'use_experimental_parser': 'False', 'invocation_command': 'dbt debug', 'no_print': 'None', 'fail_fast': 'False', 'write_json': 'True', 'introspect': 'True', 'quiet': 'False', 'log_path': 'logs', 'log_format': 'default', 'target_path': 'None', 'use_colors': 'True', 'version_check': 'True', 'log_cache_events': 'False', 'printer_width': '80', 'debug': 'False', 'partial_parse': 'True', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/home/joyce/.dbt', 'indirect_selection': 'eager', 'warn_error': 'None', 'empty': 'None', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])'}
[0m10:51:27.827280 [info ] [MainThread]: dbt version: 1.10.13
[0m10:51:27.827852 [info ] [MainThread]: python version: 3.12.3
[0m10:51:27.828137 [info ] [MainThread]: python path: /home/joyce/joyce-saas/meu-dbt-cobaia/venv/bin/python3
[0m10:51:27.828394 [info ] [MainThread]: os info: Linux-6.14.0-33-generic-x86_64-with-glibc2.39
[0m10:51:27.833018 [info ] [MainThread]: Error importing adapter: No module named 'dbt.adapters.postgres'
[0m10:51:28.035923 [info ] [MainThread]: Using profiles dir at /home/joyce/.dbt
[0m10:51:28.036334 [info ] [MainThread]: Using profiles.yml file at /home/joyce/.dbt/profiles.yml
[0m10:51:28.036787 [info ] [MainThread]: Using dbt_project.yml file at /home/joyce/joyce-saas/meu-dbt-cobaia/dbt_project.yml
[0m10:51:28.037137 [info ] [MainThread]: Configuration:
[0m10:51:28.037456 [info ] [MainThread]:   profiles.yml file [[31mERROR invalid[0m]
[0m10:51:28.037769 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m10:51:28.038055 [info ] [MainThread]: Required dependencies:
[0m10:51:28.038360 [debug] [MainThread]: Executing "git --help"
[0m10:51:28.041094 [debug] [MainThread]: STDOUT: "b"usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]\n           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]\n           [--config-env=<name>=<envvar>] <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone     Clone a repository into a new directory\n   init      Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add       Add file contents to the index\n   mv        Move or rename a file, a directory, or a symlink\n   restore   Restore working tree files\n   rm        Remove files from the working tree and from the index\n\nexamine the history and state (see also: git help revisions)\n   bisect    Use binary search to find the commit that introduced a bug\n   diff      Show changes between commits, commit and working tree, etc\n   grep      Print lines matching a pattern\n   log       Show commit logs\n   show      Show various types of objects\n   status    Show the working tree status\n\ngrow, mark and tweak your common history\n   branch    List, create, or delete branches\n   commit    Record changes to the repository\n   merge     Join two or more development histories together\n   rebase    Reapply commits on top of another base tip\n   reset     Reset current HEAD to the specified state\n   switch    Switch branches\n   tag       Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch     Download objects and refs from another repository\n   pull      Fetch from and integrate with another repository or a local branch\n   push      Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m10:51:28.042021 [debug] [MainThread]: STDERR: "b''"
[0m10:51:28.042656 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m10:51:28.043069 [info ] [MainThread]: Connection test skipped since no profile was found
[0m10:51:28.043495 [info ] [MainThread]: [31m2 checks failed:[0m
[0m10:51:28.043877 [info ] [MainThread]: Could not load dbt_project.yml
Profile loading failed for the following reason:
Runtime Error
  Credentials in profile "airflow_dbt_project", target "dev" invalid: Runtime Error
    Could not find adapter type postgres!


[0m10:51:28.044266 [info ] [MainThread]: Project loading failed for the following reason:
 project path </home/joyce/joyce-saas/meu-dbt-cobaia/dbt_project.yml> not found

[0m10:51:28.046131 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_success": false, "command_wall_clock_time": 0.28371227, "process_in_blocks": "0", "process_kernel_time": 0.101949, "process_mem_max_rss": "122608", "process_out_blocks": "24", "process_user_time": 1.401311}
[0m10:51:28.047220 [debug] [MainThread]: Command `dbt debug` failed at 10:51:28.047077 after 0.29 seconds
[0m10:51:28.047691 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fa1b22467b0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fa1b0545af0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fa1b06d8d10>]}
[0m10:51:28.048215 [debug] [MainThread]: Flushing usage events
[0m10:51:28.934977 [debug] [MainThread]: An error was encountered while trying to flush usage events
